entity-service/src/main/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationRepository.kt

```diff
@@ -340,6 +339,8 @@ class Neo4jAuthorizationRepository(
         val query =
             """
             MATCH (user)-[]->()-[right:$authTerm]->(entity:Entity { id: ${'$'}entityId })
+            WITH user, right
+            ORDER BY user.id
```

Comment:
I don't get the point of ordering items in the relationship

Reply:
it can be more interesting for the user ?

Reply:
why not, it won't hurt